### PR TITLE
Create elixir template

### PR DESCRIPTION
**Pull request by CCavalier**
I am not sure for the indentation #vim, and I have no idea of the erlang standart.

### CODE REVIEW

File: markdowns/welcome.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# Welcome!
+
+This Java template lets you get started quickly with a simple working example using Maven and JUnit. If it is your first contribution then you should have a look at the [Getting Started](https://tech.io/doc/getting-started-create-playground) document.
+
+
+The source code is on [GitHub](https://github.com/TechDotIO/java-template), please feel free to come up with proposals to improve it.
```
Comment (JeanDamien) [inline markdowns/welcome.md]: Don't forget to update template related urls :)

File: techio.yml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+plan:
+  - title: Welcome # Set the page title (only displayed if your contribution has several pages)
+    statement: markdowns/welcome.md # Set the file path to the page content
+#  - title: Another Page
+#    statement: markdowns/another-page.md
+
+
+# Settings for your programming projects and the associated Docker images
+projects:
+  java:
```
Comment (JeanDamien) [inline techio.yml]: Here too, it should be elixir or something like that

File: techio.yml [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+    root: /elixir-project # Set the root path to your Java project
```
Comment (JeanDamien) [inline techio.yml]: to your Elixir project

File: techio.yml [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+    # Set the docker image. This image runs mix tests using this syntax:
+    # @[Code Editor Title]({"stubs": ["EditorFile1.ex", "EditorFile2.ex", ...], "command": "MyMixFile.exs:10"})
+    # More details here: https://github.com/TechDotIO/java-maven3-junit4-runner
+    runner:
+      name: cgcharlotte/elixir-exunit-runner
```
Comment (JeanDamien) [inline techio.yml]: Can't we use the official Elixir image instead ? You can transform your build.sh in a buildCommand, and run your test using 'mix test testMyMixFile.exs:10' I guess, or did i miss something ?

File: markdowns/welcome.md (hunk 1 above)
Comment (Sy1v4in) [reply markdowns/welcome.md]: And the text ;-)

**Pull request merged**